sdks/client-go/pkg/analytics/google.go

```diff
@@ -0,0 +1,46 @@
+package analytics
+
+import (
+	"fmt"
+
+	"github.com/featurehub-io/featurehub/sdks/client-go/pkg/models"
+	ga "github.com/jpillora/go-ogle-analytics"
+)
+
+const (
+	defaultCategory = "FeatureHub Event"
+)
+
+// GoogleAnalyticsCollector implements the AnalyticsCollector interface:
+type GoogleAnalyticsCollector struct {
+	client       *ga.Client
+	clientID     string
+	trackingID   string
+	userAgentKey string
+}
+
+// NewGoogleAnalyticsCollector returns a GoogleAnalyticsCollector configured with the provided metadata:
+func NewGoogleAnalyticsCollector(clientID, trackingID, userAgentKey string) (*GoogleAnalyticsCollector, error) {
+	client, err := ga.NewClient(trackingID)
+	if err != nil {
+		return nil, err
+	}
+
+	return &GoogleAnalyticsCollector{
+		client:       client.ClientID(clientID),
+		clientID:     clientID,
+		trackingID:   trackingID,
+		userAgentKey: userAgentKey,
+	}, nil
+}
+
+// LogEvent generates analytics events for the given action and metadata:
+func (ac *GoogleAnalyticsCollector) LogEvent(action string, other map[string]string, featureStateAtCurrentTime map[string]*models.FeatureState) error {
+
+	// Emit an event for each feature we have:
+	for _, featureState := range featureStateAtCurrentTime {
+		event := ga.NewEvent(defaultCategory, action).Label(featureState.Key).Label(fmt.Sprintf("%s", featureState.Value)).Value(1)
```

Comment:
i  think you accidentally have two Labels here? perhaps it should be 

fmt.Sprintf("%s : %s", featureState.Key, featureState.Value)?

It is a pity this library doesn't use the batching API.

Reply:
(as in this GA library)

Reply:
No that wasn't by accident - i figured we could have arbitrary numbers of labels. Is this not the case?

Reply:
No... I also looked at the source code for that library and it just overwrites the last time you used it (which is what I expected). Rob investigated this and came up with that format as a compromise because of this issue.

Reply:
OK I fixed this up